src/databricks/labs/ucx/install.py

```diff
@@ -562,14 +562,19 @@ def _create_dashboard(self, folder: Path, *, parent_path: str | None = None) ->
         )
         metadata.display_name = f"{self._name('UCX ')} {folder.parent.stem.title()} ({folder.stem.title()})"
         reference = f"{folder.parent.stem}_{folder.stem}".lower()
-        dashboard = Dashboards(self._ws).deploy_dashboard(
-            metadata.as_lakeview(),
-            dashboard_id=self._install_state.dashboards.get(reference),
+        dashboard_id = self._install_state.dashboards.get(reference)
+        if dashboard_id is not None and "-" in dashboard_id:  # Upgrading from Redash to Lakeview
```

Comment:
Need to confirm that this works. @nfx : Can you confirm that the Redash ids always contain a "-" and Lakeview does not

Reply:
integration tests will confirm :)